src/renderer/src/components/trigger/TriggersSlice.ts

```
@@ -42,8 +40,12 @@ export const fetchTrigger = createAsyncThunk(
 
 export const fetchTriggers = createAsyncThunk(
   'triggers/fetchTriggers',
-  async () => {
-    return await IpcService.send<{ triggers: any }>(GET_MANY_TRIGGERS_CHANNEL);
+  async ({ email }: { email: string }) => {
+    const response = await IpcService.send<{ triggers: any }>(
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)